examples/nextcloud-integration/custom-mime-types-demo.json

```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "file_ext": ".zmd",
```

Comment:
Can we have a string->string map here? We can directly read the json into a map object and not have to do the parsing.